release-notes/release-notes52.md

```diff
@@ -223,6 +229,97 @@ For information on the `toolbar_mode` option, see: [User interface options - `to
 
+**Outline**
+
+* [Translations missing for new user interface options](#translationsmissingfornewuserinterfaceoptions)
+* [Floating toolbar disconnects from the toolbar in inline mode](#floatingtoolbardisconnectsfromthetoolbarininlinemode)
+* [Microsoft Internet Explorer will incorrectly auto focus the editor when TinyMCE is initialized](#microsoftinternetexplorerwillincorrectlyautofocustheeditorwhentinymceisinitialized)
+* [Incorrect positioning of inline dialogs when using sticky toolbars and the bottom toolbar](#incorrectpositioningofinlinedialogswhenusingstickytoolbarsandthebottomtoolbar)
+
+### Translations missing for new user interface options
+
+#### Issue
+
+New user interface settings do not have translations to other supported languages at this time. This includes text related to:
+
+- The new accessibility settings.
+- The new checklist menu item.
+
+#### Workaround
+
+No known workaround at the time of release.
+
+### Floating toolbar disconnects from the toolbar in inline mode
+
+#### Issue
+
+> **Note**: This issue will be resolved with the release of {{site.productname}} 5.2.1.
+
+If the editor is set to `inline: true`, and `toolbar_drawer` or `toolbar_mode` is set to `'floating'`, such as:
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  inline: true,
+  toolbar_drawer: 'floating'
+});
+```
+
+or
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  inline: true,
+  toolbar_mode: 'floating'
+});
+```
+
+The floating component of the toolbar will disconnect from the primary toolbar when page or editor is scrolled using a mouse or a scrolling touch gesture.
+
+#### Workaround
+
+Set `toolbar_drawer` or `toolbar_mode` to an alternative mode until the issue is resolved. For a list of alternative toolbar modes, see: [User interface options - `toolbar_mode`]({{site.baseurl}}/configure/editor-appearance/#toolbar_mode).
+
+### Microsoft Internet Explorer will incorrectly auto focus the editor when TinyMCE is initialized
+
+#### Issue
+
+> **Note**: This issue will be resolved with the release of {{site.productname}} 5.2.1.
+
+When a webpage containing {{site.productname}} is loaded on Microsoft Internet Explorer, the user's cursor will automatically focus on the editor when the editor is loaded.
+
+#### Workaround
+
+No known workaround at the time of release.
+
+### Incorrect positioning of inline dialogs when using sticky toolbars and the bottom toolbar
+
+#### Issue
+
+Inline dialogs, such as the _Search and Replace dialog_ and the _Accessibility Checker dialog_, will be positioned incorrectly when sticky toolbar and the bottom toolbar are used together. For example:
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  toolbar_sticky: true,
+  toolbar_location: 'bottom'
+});
+```
+
+Custom dialogs using the following API setting are also affected.
+
+```js
+editor.windowManager.open(spec, {inline: 'toolbar'})
```

Comment:
We don't actually document how to open inline dialogs, so maybe we shouldn't be saying this? We've also just referred to this as "inline dialogs" in the 5.1 release notes, so maybe just say something like this instead?

> Custom inline dialogs are also affected.


Note: We probably should have documented how to do inline dialogs though, but that's a task for another day.

Reply:
dropped the codeblock and changed the text to match the suggestion